docs/tables/turbot_policy_value.md

```diff
@@ -0,0 +1,93 @@
+# Table: turbot_resource
+
+Resources in Turbot represent cloud configuration items such as users,
+networks, servers, etc.
+
+It is recommended that queries to this table should include (usually in the `where` clause) at least one
+of these columns: `id`, `resource_type_id`, `resource_type_uri` or `filter`.
+
+A Policy Value is the effective policy setting on an instance of a resource type. Every resource that is targeted by a given policy setting will have its own value for that policy, which will be the resultant calculated policy for the "winning" policy in the hierarchy.
+
+Policy settings are inherited through the resource hierarchy, and values for a resource are calculated according to policy settings at or above it in the resource hierarchy. For example, a policy setting at the Turbot level will be inherited by all resources below.
+
+It is recommended that queries to this table should include (usually in the `where` clause) at least one
+of these columns: `state`,`policyTypeId`, `resource_type_id`, `resource_type_uri` or `filter`.
+
+## Examples
+
+### List policy values by policy type id
+
+```sql
+select
+  id,
+  state,
+  is_default,
+  is_calculated,
+  policy_type_id,
+  type_mod_uri
+from
+  turbot_policy_value
+where
+  policy_type_id = 123456789;
```

Comment:
Can you please use a value here that looks like a policy type ID? I understand it won't match to a real resource ID when running it, but please see other table docs as an example

Reply:
Added the `policy_type_id` value